doc/io.rst

```diff
@@ -440,6 +440,17 @@ supported by netCDF4-python: 'standard', 'gregorian', 'proleptic_gregorian' 'nol
 By default, xarray uses the 'proleptic_gregorian' calendar and units of the smallest time
 difference between values, with a reference time of the first time value.
 
+Coordinates
+...........
+
+You can control the ``coordinates`` attribute written to disk by specifying ``encoding["coordinates"]``.
```

Comment:
I think it would be good to explicitly state that the we're talking about `DataArray.encoding`. The notion of the _global coordinate_ makes me think of a `Dataset.encoding['coordinate']` attribute. Perhaps you can clarify here (and in the whats new note).

Reply:
:+1: